src/schema/artwork/index.js

```diff
@@ -122,6 +122,88 @@ export const artworkFields = () => {
     category: {
       type: GraphQLString,
     },
+    attribution_class: {
+      type: GraphQLString,
+      description: "Attribution class as classification name",
+      resolve: artwork => {
+        if (artwork.attribution_class) {
+          return capitalizeFirstCharacter(artwork.attribution_class)
+        }
+      },
+    },
+    attribution_class_hover: {
+      type: GraphQLString,
+      description: "Descriptive sentance used as companion for attribution class name display",
+      resolve: artwork => {
+        switch (artwork.attribution_class) {
+          case "editioned multiple": return "High quantity editions, without direct artist involvement."
```

Comment:
It might be cleaner to have a JSON file here with the info, similar to the spec above.

We already do that for a few things, here's an example file: https://github.com/artsy/metaphysics/blob/master/src/lib/auction_records_trusted.json

Then you can just `require(...)` it.